cunumeric_cpp.cmake

```diff
@@ -355,6 +355,7 @@ target_link_libraries(cunumeric
           $<TARGET_NAME_IF_EXISTS:conda_env>
           $<TARGET_NAME_IF_EXISTS:CUDA::cufft>
           $<TARGET_NAME_IF_EXISTS:CUDA::cublas>
+          $<TARGET_NAME_IF_EXISTS:CUDA::curand>
```

Comment:
it looks like we don't need to link against libcurand (see https://github.com/nv-legate/cunumeric/pull/541). can you try to build things without this line and see if that works?

Reply:
Sure, I can remove it. Conda prints a warning that the `libcurand` package isn't used when I build the conda package locally, but I'm fine ignoring that message.